### PR TITLE
ADEN-5037 fix intestitial tests

### PR DESCRIPTION
**Pull request by bulaszenko**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsInterstitialObject.java [hunk 1]
```diff
@@ -123,7 +102,7 @@ public void closeInterstitial() throws InterruptedException {
   }
 
   public void verifyInterstitialIsClosed() {
-    waitForElementNotVisibleByElement(interstitialAdWrapper);
+    waitForElementNotVisibleByElement(interstitialAd);
   }
 
   private boolean isSizeCorrect(int actualSize, int expectedSize) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsInterstitialObject.java]: MAJOR Remove this unused private "isSizeCorrect" method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java [hunk 2]
```diff
@@ -762,69 +762,23 @@ private AdsDataProvider() {
...
-            true
-        },
-        {
-            "project43",
-            "SyntheticTests/Interstitial/NotScalable",
-            new Dimension(800, 500),
-            new Dimension(300, 258),
-            false
+            new Page("project43", "SyntheticTests/Slots/InvisibleHighImpact/Interstitial"),
+            "#INVISIBLE_HIGH_IMPACT_2 .provider-container iframe",
```
Comment (fraszczakszymon) [inline src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java]: I believe we can have the same selector on mobile and desktop. Please use:
`.invisible-high-impact-wrapper.interstitial .provider-container iframe`

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsInterstitial.java [hunk 3]
```diff
@@ -14,48 +19,30 @@
...
-      String wikiName,
-      String article,
-      Dimension pageSize,
-      Dimension adSize,
-      boolean shouldAdBeScaled
-  ) throws InterruptedException {
-    String url = urlBuilder.getUrlForPath(wikiName, article);
-    String testedPage = urlBuilder.appendQueryStringToURL(url, "highimpactslot=1");
-    AdsInterstitialObject adsInterstitial = new AdsInterstitialObject(driver, testedPage, pageSize);
+  private void testInterstitial2(Page page, String selector, Dimension adSize) throws InterruptedException {
```
Comment (fraszczakszymon) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsInterstitial.java]: Rename to `testInterstitial` please

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsInterstitial.java [hunk 4]
```diff
@@ -14,48 +19,30 @@
...
   )
-  public void adsInterstitialAdScaledOasis(
-      String wikiName,
-      String article,
-      Dimension pageSize,
-      Dimension adSize,
-      boolean shouldAdBeScaled
-  ) throws InterruptedException {
-    testInterstitial(wikiName, article, pageSize, adSize, shouldAdBeScaled);
+  public void adsInterstitialAdScaledOasis(Page page, String selector, Dimension adSize) throws InterruptedException {
```
Comment (fraszczakszymon) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsInterstitial.java]: Is it still scaled?

**Pull request merged**